src/digodoc_common/main.ml

```diff
@@ -10,7 +10,8 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(* options that are modified only using digodoc *)
-let sources = ref true
 
-let with_header = ref false
\ No newline at end of file
+(* If you delete or rename this file, you should add
+   'src/digodoc_common/main.ml' to the 'skip' field in "drom.toml" *)
+
+let main () = Printf.printf "Hello world!\n"
```

Comment:
I believe this is a mistake.

Reply:
This library was generated automatically with library skeleton by _drom_. But I confirm, this file doesn't have any sens